gammapy/datasets/map.py

```diff
@@ -2685,28 +2707,47 @@ def update(self, exposure, psf, edisp, geom, mask):
                     geom = geom.to_wcs_geom()
 
                 self.psf = psf.get_psf_kernel(
-                    position=self.model.position,
-                    geom=geom,
-                    containment=PSF_CONTAINMENT
+                    position=self.model.position, geom=geom, containment=PSF_CONTAINMENT
                 )
+            self._psf_r68_min = psf.containment_radius(
```

Comment:
Is this extra handling of the `_psf_r68_min` required? For point sources we don't need oversampling, because the model is always correctly integrated on the normal evaluation grid.